README.md

```diff
@@ -1 +1,186 @@
-# hratx53
\ No newline at end of file
+# Team Papercuts
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/hratx-blue-ocean/Papercuts">
+    <img src="Docs/readMeImage/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">Papercuts Bookstore</h3>
+
+  <p align="center">
+    OUR MISSION: An application to connect your community through reading and sharing books. 
+    <br />
+    <a href="https://github.com/hratx-blue-ocean/Papercuts"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/hratx-blue-ocean/Papercuts">View Demo</a>
+    ·
+    <a href="https://github.com/hratx-blue-ocean/Papercuts/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/hratx-blue-ocean/Papercuts/issues">Request Feature</a>
+  </p>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgements">Acknowledgements</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+#### Applicatin login with proper authentication
+
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
+
+Project Description
+
+Key Features
+
+- User authentication
+- Display the book of the month
+- Explore trending books from the store
+- View user libary and purchased book
+- Explore user libary and
+- Connect and add friends
+- Create bookclub and bookclub event
+- Email notification for bookclub event
+
+### Built With
```

Comment:
Can we use logos for our tech stack like
![image](https://user-images.githubusercontent.com/64542531/108141141-cf41ef00-7088-11eb-9c57-2ea309457cba.png)

Reply:
Ok, I am adding it to there